packages/console/src/screens/app-builder/panels/explorer/manager/LayerManagerProvider.tsx

```diff
@@ -3,7 +3,7 @@ import React from "react";
 import { useMethods } from "../../../../../craft-utils";
 import { LayerEventContextProvider } from "../events";
 import { LayerOptions } from "../interfaces";
-import { DefaultLayer } from "../layers";
+import { DefaultLayer } from "../layers/index";
```

Comment:
This seems a little odd. Shouldn't TS/Webpack pickup `../layers/index.ts` via `../layers`?